*** pkg/i3-gnome-flashback	2020-06-10 19:18:03.412477710 -0400
--- i3-gnome-flashback	2020-06-10 19:32:32.776068488 -0400
***************
*** 63,81 ****
  if [ ! -f "$UPDATE_FLAG_FILE" ]; then
      echo "Executing Regolith session configuration script."
  
-     GNOME_FLASHBACK_VERSION_STR=$(dpkg -s gnome-flashback | grep '^Version:' | cut -d' ' -f 2 | cut -d'-' -f 1)
-     GNOME_FLASHBACK_VERSION="${GNOME_FLASHBACK_VERSION_STR//.}"
- 
-     if [ $GNOME_FLASHBACK_VERSION -le 3340 ]; then
-         gsettings set org.gnome.desktop.background show-desktop-icons false
-         gsettings set org.gnome.gnome-flashback desktop-background true
-     else
          gsettings set org.gnome.gnome-flashback desktop false
          gsettings set org.gnome.gnome-flashback root-background true
-     fi
  
-     GNOME_FLASHBACK_VERSION_STR=$(dpkg -s gnome-flashback | grep '^Version:' | cut -d' ' -f 2 | cut -d'-' -f 1)
-     GNOME_FLASHBACK_VERSION="${GNOME_FLASHBACK_VERSION_STR//.}"
  
      # Backing existing GNOME settings if possible
      if [ ! -f "$BACKUP_GNOME_SETTINGS" ]; then
--- 63,71 ----
***************
*** 89,101 ****
          echo "gsettings set org.gnome.desktop.wm.preferences button-layout $(gsettings get org.gnome.desktop.wm.preferences button-layout)" >>$BACKUP_GNOME_SETTINGS
          echo "gsettings set org.gnome.gnome-flashback status-notifier-watcher $(gsettings get org.gnome.gnome-flashback status-notifier-watcher)" >>$BACKUP_GNOME_SETTINGS
  
-         if [ $GNOME_FLASHBACK_VERSION -le 3340 ]; then
-             echo "gsettings set org.gnome.desktop.background show-desktop-icons $(gsettings get org.gnome.desktop.background show-desktop-icons)" >>$BACKUP_GNOME_SETTINGS
-             echo "gsettings set org.gnome.gnome-flashback desktop-background $(gsettings get org.gnome.gnome-flashback desktop-background)" >>$BACKUP_GNOME_SETTINGS
-         else
              echo "gsettings set org.gnome.gnome-flashback desktop $(gsettings get org.gnome.gnome-flashback desktop)" >>$BACKUP_GNOME_SETTINGS
              echo "gsettings set org.gnome.gnome-flashback root-background $(gsettings get org.gnome.gnome-flashback root-background)" >>$BACKUP_GNOME_SETTINGS
-         fi
      fi
  
      # Disable built-in notification system for Rofication
--- 79,86 ----
