--- a/usr/bin/regolith-look	2020-12-23 14:49:59.183459483 -0500
+++ b/usr/bin/regolith-look	2020-12-23 14:51:31.575970850 -0500
@@ -55,10 +55,11 @@
     if [[ ! -z $WALLPAPER_FILE ]]; then
         gsettings set org.gnome.desktop.background picture-uri "file://$(eval echo $WALLPAPER_FILE)"
     fi
-
-    UPDATE_TERM_FLAG=$(xrescat gnome.terminal.update true)
-    if [[ "$UPDATE_TERM_FLAG" == "true" && -f "/usr/bin/create-regolith-term-profile" ]] ; then
-        /usr/bin/create-regolith-term-profile
+    if hash gnome-terminal 2>/dev/null; then # Check if gnome-terminal is in the $PATH
+        UPDATE_TERM_FLAG=$(xrescat gnome.terminal.update true)
+        if [[ "$UPDATE_TERM_FLAG" == "true" && -f "/usr/bin/create-regolith-term-profile" ]] ; then
+            /usr/bin/create-regolith-term-profile
+        fi
     fi
 
     # restart i3-wm after merging Xresources
